### ISSUE TITLE
AuthenticationDatabase for MongoDB Auth

### ISSUE THREAD

**Issue by Kolano** [cuckoosandbox/cuckoo#1712]
It seems in order to log in to MongoDB via an account from another DB with cross-DB permissions now requires the "authSource" database be specified.

`MongoClient(mongodb://multiDBaccount:password@127.0.0.1/?authSource=admin).cuckoo`

Can a field to specify/account for such be added to the reporting.conf in addition to username / password?

**Comment by jbremer** [comment]
Yes, I guess. Is there a default `authSource` table name or something?

**Comment by Kolano** [comment]
A bit new with this but I believe the default is blank, which authenticates against the users registered with the referenced DB.
